scripts/split_dataset.py

```diff
@@ -33,6 +33,20 @@
         "traffic_sign_90",
         "vehicle"]
 
+SUPER_CATEGORY = {
+        "RSVD": "RSVD",
```

Comment:
Note: RSVD stands for reserved so that the class IDs remain 1-based to be consistent with the original COCO format.